index.bs

```diff
@@ -2107,20 +2126,16 @@ an |ownership type|, a |serialization internal map|, a |realm| and a |session|:
              1. If |shadow root| is null, let |serialized shadow| be null.
                 Otherwise run the following substeps:
 
-               1.  Let |child depth| be |max depth| - 1 if |max depth| is not
-                  null, or null otherwise.
-
                1. Let |serialized shadow| be the result of
-                  [=serialize as a remote value=] with |shadow root|, |child depth|,
-                  false, |ownership type|, |serialization internal map|,
+                  [=serialize as a remote value=] with |shadow root|,
+                  |serialization options|, |ownership type|, |serialization internal map|,
                   |realm|, and |session|.
 
-                Note: this means the <code>handle</code> for the shadow root
-                will be serialized irrespective of whether the shadow is open or closed,
-                but no properties of the node will be returned.
-
              1. Set |serialized|["<code>shadowRoot</code>"] to |serialized shadow|.
```

Comment:
Since you're on the topic, maybe you could fix [this issue](https://w3c.github.io/webdriver-bidi/#issue-e8085260)?
```suggestion
             1. If |serialized shadow| is not null, set |serialized|["<code>shadowRoot</code>"] to |serialized shadow|.
```